Lib/zlib.py

```diff
@@ -19,6 +19,7 @@
 import struct
 import sys
 from cStringIO import StringIO
+from org.python.core.util import StringUtil
```

Comment:
It is not necessary to use `StringUtil.toBytes` explicitly. Now we understand why it is necessary to call the 3-argument form of `update` we can be assured that Jython will convert `str` to `byte[]` by itself, and with less copying.